tests/e2e/specs/myOrders_spec.js

```diff
@@ -0,0 +1,48 @@
+describe('my orders', () => {
+  const customer = {
+    firstName: 'Charlie',
+    lastName: 'Bucket',
+    email: 'charlie.bucket+ci@commercetools.com',
+    password: 'p@ssword',
+  };
+
+  const cartDraft1 = {
+    customerEmail: 'charlie.bucket+ci@commercetools.com',
+    currency: 'EUR',
+    lineItems: {
+      sku: 'M0E20000000DPZ0',
+    },
+    shippingAddress: {
+      country: 'DE',
+    },
+  };
+
+  const cartDraft2 = {
+    customerEmail: 'charlie.bucket+ci@commercetools.com',
+    currency: 'EUR',
+    lineItems: {
+      sku: 'M0E20000000DLPH',
+    },
+    shippingAddress: {
+      country: 'DE',
+    },
+  };
+
+  before(() => {
+    cy.login(customer);
+  });
+
+  it('shows my orders', () => {
+    cy.createMyOrder(cartDraft1);
+    cy.createMyOrder(cartDraft2);
+    cy.get('[data-test=my-orders-button]').click();
+    cy.reload();
+    cy.url().should('include', '/user/orders');
+    cy.get('[data-test=order-date]')
+      .should('exist');
+    cy.get('[data-test=total-price]')
+      .contains('368,75');
+    cy.get('[data-test=total-price]')
+      .contains('372,50');
```

Comment:
If the price is `23.372,50`, would it fail this test?
Also, if there are two orders, which one are you testing here?
Could you also check that there are actually 2 orders in the list?

Reply:
I'm adding exact products to the orders and I know their price so I check if this products were added by checking their price. But yes I could also check if there are 2 orders in the list.